tests/baselines/reference/isolatedDeclarationErrorsClassesExpressions.errors.txt

```diff
@@ -0,0 +1,42 @@
+isolatedDeclarationErrorsClassesExpressions.ts(1,20): error TS9022: Inference from class expressions is not supported with --isolatedDeclarations.
+isolatedDeclarationErrorsClassesExpressions.ts(15,26): error TS9013: Expression type can't be inferred with --isolatedDeclarations.
+isolatedDeclarationErrorsClassesExpressions.ts(15,26): error TS9021: Extends clause can't contain an expression with --isolatedDeclarations.
+isolatedDeclarationErrorsClassesExpressions.ts(19,25): error TS9022: Inference from class expressions is not supported with --isolatedDeclarations.
+isolatedDeclarationErrorsClassesExpressions.ts(19,35): error TS9022: Inference from class expressions is not supported with --isolatedDeclarations.
+
+
+==== isolatedDeclarationErrorsClassesExpressions.ts (5 errors) ====
+    export const cls = class {
+                       ~~~~~
+!!! error TS9022: Inference from class expressions is not supported with --isolatedDeclarations.
+!!! related TS9027 isolatedDeclarationErrorsClassesExpressions.ts:1:14: Add a type annotation to the variable cls.
+        foo: string = "";
+    }
+    
+    
+    function id<T extends new (...a: any[]) => any>(cls: T) {
+        return cls;
+    }
+    
+    
+    export class Base {
+    
+    }
+    
+    export class Mix extends id(Base) {
+                             ~~~~~~~~

```

Comment:
The error here looks like a duplication, do you think this should be an issue? I personally feel that this is not good for UX.

Reply:
Removed the redundant error